docs/how_to/enabling_access_to_vmpooler.md

```diff
@@ -0,0 +1,18 @@
+# Enabling access to the vmpooler VMs during an acceptance test run
+
+If you are running acceptance tests for Beaker that utilise vmpooler, you will need to run an SSH agent and add the acceptance RSA key prior to running the tests
+
+To load the SSH agent and add the acceptance key, run the following:
+
+~~~bash
+eval `ssh-agent`
+ssh-add ~/.ssh/id_rsa-acceptance
+~~~
+  
+Now run acceptance tests that use the vmpooler:
+
+~~~bash
+bundle exec rake test:subcommands
```

Comment:
We do want to use rake tasks as shortcuts to all of the common testing paths in beaker, so they're easier to reproduce for developers. I do think the wording could use some work to make that clearer though. Maybe:

    A common example of this functionality for beaker developers is
    in testing subcommands. There, we setup multiple SUTs that need
    to communicate between themselves. To run our subcommand
    testing to verify that you have agent forwarding setup correctly,
    run this testing rake task: